content/authors/elaineliwang/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+# Display name
+title: Li Wang
+
+# Username (this should match your github username)
+authors:
+- elaineliwang
+
+superuser: false
+
+# Role/position e.g. Graduate student
+role:  GRADUATE STUDENT
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name:  University of British Columbia
+  url:  http://www.math.ubc.ca/~lwang/
+
+# A list of your interests, add as many as you want
+interests:
+-  Partial Differential Equations
+-  
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: B.Sc in Mathematics
```

Comment:
I just made up a degree here (a B.Sc from UBC), could you edit it to something relevant to you?

Reply:
Thanks! @ianabc it is fixed!